docs/docusaurus/versioned_docs/version-0.18/resources/get_support.md

```diff
@@ -4,9 +4,26 @@ title: 'Get support'
 description: Learn what resources are available to help you resolve issues with GX Cloud and GX OSS.
 ---
 
-Great Expectations (GX) relies on an active user community for its continued success. Input from users like you helps GX and the community build its collective knowledge and improve GX Cloud and GX OSS. Knowing how and where to seek support will help you get the answers you need to resolve issues.
+## Where to get support
```

Comment:
![image](https://github.com/great-expectations/great_expectations/assets/104205257/2f6be469-2299-479e-847f-c5c488f22e1d)

Add one or two sentences that describe the content that follows the heading and its purpose. Alternatively, remove the existing H@ heading and promote the H3 headings. For help with heading structure, see [Heading and title format](https://developers.google.com/style/headings#heading-and-title-format).